### ISSUE TITLE
Notice of Support Availability and Expectations

### ISSUE THREAD

**Issue by Mike-EEE** [ExtendedXmlSerializer/home#383]
Wow, as I write this I see that we have acquired 160 stars here in the ExtendedXmlSerializer.  As [I recently regaled](https://twitter.com/MikeTripleE/status/1239823209593266176), 150 stars isn't really anything, but it's not nothing, either.  Thank you so much to everyone who has supported and contributed to this project!

With that said, I wanted to make a bit of a status update about where I am in this project, personal energies, and what can be expected going forward.

I climbed on board here in November of 2016, to help out with a few bugs that I thought were an "easy" fix.  Well, one thing led to another and wouldn't you know it, I rewrote the whole thing from scratch and that is where we sit today. 🤷‍♂️ 😅

At the time, I had something to prove, mostly [because I felt Xaml was dying](https://github.com/dotnet/runtime/issues/16243) and I couldn't let that happen.  The other issue was that there wasn't really any good Xml serializers out there that did what I wanted and @WojciechNagorski's approach was appealing to me and showed promise.  To me, `System.Xaml` was and still is the ideal .NET serialization model and is what I used to model ExtendedXmlSerializer v2.

Well, a funny thing happened when v2 was "done."  It wasn't actually done. 😁  Over 300 issues later (and counting) you can say that it's an on-going process.

It's been a good learning lesson to me: you can go out and try to prove something, but you had better be ready to prove you can face the biggest challenge of them all -- time!

Up to this point, I have been pretty lucky and have been able to provide my time to helping issues to this repository.  It has been a good run now for nearly 3.5 years.

Unfortunately, that is coming to a close.  I am now focused on a startup endeavor that will be taking up my cycles in the near future and (hopefully) beyond.

**What that means is that for ExtendedXmlSerializer, my time will now be relegated to assisting with small matters and quick bug fixes.**  In addition to these, I will continue to answer questions and provide code samples to assist with onboarding those who are testing out the tires here to see if it's a fit for their projects.

Anything beyond this level of effort will be left for the community to handle.  These will be demarked with the [`help-wanted` tag](https://github.com/ExtendedXmlSerializer/home/labels/help%20wanted) for those brave enough to take them up and submit a PR.  My hope is that we can summon up a few individuals who are more aligned with open source principles and take over this project in the long run.

It will be tough, though, as it will mean having to deal with my code. 😅  But such is the nature of the reality we find ourselves in, currently.

So with that, I thank you all again for all of your support, and for actually using this sucker.  It means a lot to me!  It's actually the most successful thing I have accomplished in a while.  And of course, it would not be possible at all if @WojciechNagorski hadn't put the effort in the first place to make it a reality!  I cannot thank you enough for not only your efforts but also camaraderie and teamwork. We make a good international team. 👍 

To summarize in end, I have learned a lot in these past several years and take joy in knowing that something I helped build works in some capacity out there.

Please feel free to chime in with any questions/concerns below.  I will be pinning this issue for visibility.

Thanks again (and again!) to all!

(BTW, if interested, you can follow along with my next project [here](https://starbeam.one/).)

**Comment by xsoheilalizadeh** [comment]
Hi @Mike-E-wins, thanks for spending your valuable time on this amazing library, I appreciate it and I hope you to be a success in your next project.

**Comment by Mike-E-wins** [comment]
Thank you so much for that @xsoheilalizadeh!  That really made my day. 👍

**Comment by Mike-E-angelo** [comment]
> (BTW, if interested, you can follow along with my next project [here](https://starbeam.one/).)

FWIW I announced the preview for this product yesterday:
https://blog.starbeam.one/2022/02/announcing-mvp-alpha-preview/

Note that ExtendedXmlSerializer can be seen here specifically for the Preview button functionality:
https://youtu.be/3OJ2VOCDqoQ?t=264

In this case, I had to serialize a DTO to send to the PreviewService, and this DTO used several inheriting classes.  In order to make this work in JSON it required a bunch of configuration, and I felt it would be a poor use of time since I had basically put together a serializer that does all this automatically :)

I also mention ExtendedXmlSerializer (along with half of NuGet I feel 😅) here:
https://alpha.starbeam.one/about/acknowledgements

And finally, a little shout out to @WojciechNagorski and this project here:
https://youtu.be/B5bmxlRtvw8?t=850
